### ISSUE TITLE
Cannot read notes in powerpoint 2016

### ISSUE THREAD

**Issue by Neurrone** [nvaccess/nvda#7937]
### Steps to reproduce:
1. Open a presentation with slide notes.
2. NVDA continually errors, and doesn't read anything.

### Expected behavior:
Notes should be read correctly.

### Actual behavior:

```sh
error executing event: gainFocus on <appModules.powerpnt.NotesTextFrame object at 0x06FA3C90> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 152, in executeEvent
  File "eventHandler.pyc", line 92, in __init__
  File "eventHandler.pyc", line 100, in next
  File "NVDAObjects\__init__.pyc", line 951, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 839, in reportFocus
  File "speech.pyc", line 394, in speakObject
  File "documentBase.pyc", line 24, in makeTextInfo
  File "textInfos\offsets.pyc", line 303, in __init__
  File "appModules\powerpnt.pyc", line 894, in _getStoryLength
  File "comtypesMonkeyPatches.pyc", line 35, in new__getattr__
  File "comtypes\client\lazybind.pyc", line 168, in __getattr__
  File "comtypes\automation.pyc", line 729, in _invoke
COMError: (-2147352567, 'Exception occurred.', (u'The specified value is out of range.', None, None, 0, None))
```

### System configuration:
NVDA version: Version: master-14798,e6292ba9
NVDA Installed or portable: installed

Other information:
Windows version: windows 10 1709 10.0.16299.192

Name and version of other software in use when reproducing the issue:
Office 365 Microsoft® PowerPoint® 2016 MSO (16.0.8827.2131) 32-bit

### Other questions:

Does the issue still occur after restarting your PC? Yes

Have you tried any other versions of NVDA? No

**Comment by PratikP1** [comment]
I'm not able to replicate this behavior with several presentations I've tried. I'm attaching a sample test presentation for review.
* NVDA version next-14799,5410e284
* Microsoft® PowerPoint 2016 MSO (16.0.8431.2110)
*Windows 10 1709 build 17074.1002

[test presentation with notes.pptx](https://github.com/nvaccess/nvda/files/1657048/test.presentation.with.notes.pptx)

**Comment by leonardder** [comment]
@Neurrone, Could you please test this with NVDA 2017.4?

**Comment by Neurrone** [comment]
@PratikP1 The slides you posted doesn't error for me. However, when I paste a slide of a presentation that does this into [this set of test slides,](https://www.dropbox.com/s/wsjzi1hkzsxn1ni/test.presentation.with.notes.pptx?dl=1) it happens.

**Comment by PratikP1** [comment]
@Neurrone, have you been able to replicate this behavior with other presentations? I can't determine what is causing the actual issue. In other words, it's not certain whether NVDA needs some kind of a fix or the problem lies  in the composition of that particular presentation. Do you know if the presentation was created in Office 2016?

**Comment by ehollig** [comment]
@Neurrone, can you address the question in https://github.com/nvaccess/nvda/issues/7937#issuecomment-360058486? Specifically, are you testing this in Powerpoint 2016, and was this presentation created in Office 2016?
Also, please provide exact detailed steps to reproduce, including what you did, what you expected to happen and what happened instead.
1. Open [this set of test slides,](https://www.dropbox.com/s/wsjzi1hkzsxn1ni/test.presentation.with.notes.pptx?dl=1) with Powerpoint 2016.
2. Press F5 for slideshow view.
3. Press CTRL+Shift+S to view the notes for the slides.
4. Press space to advance through the presentation. No errors are present, at least for me using latest Master.

**Comment by Neurrone** [comment]
@ehollig @PratikP1 unfortunately I can't easily find out what powerpoint version was used to create these slides, but I do remember encountering this issue before, though can't find the other slides. Yes, this was opened in Powerpoint 2016.

1. Open the presentation, I had to alt-tab out and back in before NVDA would announce that the presentation was loaded.
2. Press down arrow twice to go to the third slide. Press f6 repeatedly till you land on the notes pane. NVDA errors as described.

**Comment by Neurrone** [comment]
@ehollig the steps you described using presentation view works for me as well without error - its only erroring when you try to edit the notes on the third slide.

**Comment by Adriani90** [comment]
Still reproducible with NVDA 2025.1.2 and Powerpoint 365 (Version 2505 Build 16.0.18827.20102) 64 Bit.
